src/components/util/resolve-cell-styles.js

```diff
@@ -1,7 +1,31 @@
-import merge from "lodash.merge";
+/* eslint-disable no-magic-numbers */
 import { mergeStyles } from "radium/lib/merge-styles";
+import Fraction from "fraction.js";
```

Comment:
How big/necessary is `Fraction` compared to something more naive? What are the edge cases this lib solves for us?